src/store/lokapi.ts

```diff
@@ -46,6 +47,39 @@ export function lokapiStoreFactory(lokApiService: any, gettext: any) {
         }
         dispatch("setupAfterLogin")
       },
+      async signup(
```

Comment:
I'm not sure I understand what we get by adding this in the store.  Why not call it directly in the only place you need it ?